src/components/Video/index.js

```diff
@@ -74,6 +74,7 @@ class Video extends Component<Props, State> {
   }
 
   handleRecordingStart = () => {
+    console.log('handleRecordingStart!')
```

Comment:
`console.log` - ignore if this intentional.

Reply:
Oops. Got left behind during clean up.